### ISSUE TITLE
Fix compatibility on Windows (OpenCV imread method)

### ISSUE THREAD

**Issue by fxred** [fxred/ASVR#11]
Cannot retrieve images through OpenCV using special characters such as ö:
![image](https://github.com/fxred/ASVR/assets/90219532/d505c4cf-3c19-48a1-adbd-5dbfa4416310)
![image](https://github.com/fxred/ASVR/assets/90219532/a53f53db-2531-466c-90e4-1c47e4e2b574)
Works well on Linux though

**Comment by fxred** [comment]
- Probably it would be better to rename the original file using RegEx and remove any special characters that could break the imread method, save it elsewhere and use that temporary file for the resizing.

**Comment by fxred** [comment]
Resolved on https://github.com/fxred/ASVR/commit/9b3ab381ed5e0865f11ff85ba9dc1f38f3998b71. This issue couldn't be fixed without changing the way OpenCV reads the file, as that problem is not bound to the filename, but also its filepath.
TL;DR is: I swapped the imread and imwrite methods to imdecode and imencode respectively, as well as saving and reading the image data to and from a NumPy array.
